docs/machines/note-olcf-spack-dir.rst

```diff
@@ -0,0 +1,19 @@
+
+.. note::
+  
+   Consider also configuring spack to use gpfs scratch space (i.e. ``$MEMBERWORK``)
+   when building packages, rather than the home filesystem which tends to have
+   problems with high workload tasks:
+
+   .. code-block:: sh
+
+      $ mkdir -p /gpfs/alpine/scratch/$USER/spack-stage
+
+      and add the following to ``~/.spack/config.yaml``:
```

Comment:
remove indentation to move this out from the `code-block`

Reply:
Fixed.